<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(219, 219, 219);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="importLoops">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(213, 213, 213);
	color:black;
	border-style:solid;
	border-width: 1px; 
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: white;
}
</string>
    </property>
    <property name="text">
     <string>Import Loops</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteLoop">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>85</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(213, 213, 213);
	color:black;
	border-style: solid;
	border-width: 1px; 
	border-radius: 10px;
}

QPushButton:pressed {
	background-color: white;
}</string>
    </property>
    <property name="text">
     <string>Delete Loop</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exportMaster">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>114</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(168, 176, 255);
	color:black;
	border-style: solid;
	border-width: 1px; 
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: rgb(255, 169, 251);
}</string>
    </property>
    <property name="text">
     <string>Export Master</string>
    </property>
   </widget>
   <widget class="QPushButton" name="replace">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>335</y>
      <width>160</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(168, 176, 255);
	color:black;
	border-style: solid;
	border-width: 1px; 
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: rgb(255, 169, 251);
}</string>
    </property>
    <property name="text">
     <string>Replace</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectMaster">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(213, 213, 213);
	color:black;
	border-style: solid;
	border-width: 1px; 
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: white;
}</string>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectLoop1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(213, 213, 213);
	color:black;
	border-style: solid;
	border-width: 1px; 
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: white;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectLoop2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(213, 213, 213);
	color:black;
	border-style: solid;
	border-width: 1px; 
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: white;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leftArrow">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>355</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(213, 213, 213);
	color:black;
	border-style: solid;
	border-width: 1px; 
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: white;
}</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightArrow">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>355</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(213, 213, 213);
	color:black;
	border-style: solid;
	border-width: 1px; 
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: white;
}</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>450</y>
      <width>541</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(213, 213, 213);
	color:black;
	border-style: solid;
	border-width: 1px; 
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: white;
}</string>
    </property>
    <property name="text">
     <string>â–¶</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>306</x>
      <y>50</y>
      <width>140</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(66, 69, 183);
border-width: 3px;        
border-style: solid;
border-radius: 25px;
color:black;

</string>
    </property>
    <property name="text">
     <string>LoopStorm</string>
    </property>
   </widget>
   <widget class="QLabel" name="loop0">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>150</y>
      <width>541</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
border-style: outset;
border-width: 1px; 
border-radius: 2px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>Master Loop</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="loop1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>200</y>
      <width>541</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
border-style: outset;
border-width: 1px; 
border-radius: 2px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>Loop 1</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="loop2">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>250</y>
      <width>541</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
border-style: outset;
border-width: 1px; 
border-radius: 2px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>Loop 2</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sliceText">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>327</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Master Slice Selector</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="rightArrow_2">
    <property name="geometry">
     <rect>
      <x>613</x>
      <y>355</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(213, 213, 213);
	color:black;
	border-style: solid;
	border-width: 1px; 
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: white;
}</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="sliceText_2">
    <property name="geometry">
     <rect>
      <x>538</x>
      <y>327</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Slave Slice Selector</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="leftArrow_2">
    <property name="geometry">
     <rect>
      <x>533</x>
      <y>355</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(213, 213, 213);
	color:black;
	border-style: solid;
	border-width: 1px; 
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: white;
}</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="putBack">
    <property name="geometry">
     <rect>
      <x>294</x>
      <y>385</y>
      <width>160</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:rgb(168, 176, 255);
	color:black;
	border-style: solid;
	border-width: 1px; 
	border-radius: 5px;
}

QPushButton:pressed {
	background-color: rgb(255, 169, 251);
}</string>
    </property>
    <property name="text">
     <string>Put Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
